set.mm

```diff
@@ -638670,6 +638670,162 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       WHAVKVLVNCAGDVEZABWEWFVBWGVFVGVHAVSWNABVRWEWFAVKVLCWOWGVIVJVGTTT $.
   $}
 
+  ${
+    $d A x $.  $d A y $.  $d B x $.  $d B y $.  $d ph x $.  $d ph y $.
+    dvrelog2.1 $e |- ( ph -> A e. RR ) $.
+    dvrelog2.2 $e |- ( ph -> B e. RR ) $.
+    dvrelog2.3 $e |- ( ph -> 0 < A ) $.
+    dvrelog2.4 $e |- ( ph -> A <_ B ) $.
+    dvrelog2.5 $e |- F = ( x e. ( A [,] B ) |-> ( log ` x ) ) $.
+    dvrelog2.6 $e |- G = ( x e. ( A (,) B ) |-> ( 1 / x ) ) $.
+    $( The derivative of the logarithm, ~ ftc2 version.  (Contributed by
```

Comment:
What does "ftc2 version" mean ?

Reply:
https://us.metamath.org/mpeuni/ftc2.html

Reply:
comparing the statements, dvrelog2 considers the derivative of the logarithm on a specific range, in contrast to dvrelog

So dvrelog2 is a variant of dvrelog where you can apply ftc2, which is on a specific range

Reply:
Actually now that I think about it it explains why I made the first version like it is. It was like that because my first integration proof used https://us.metamath.org/mpeuni/itgparts.html, which was a colossal pain in the ass to get the hypotheses right.

After checking out most derivative maps to builders I have figured out that most operate on an open set.

My proposed solution would be to change itgparts to use open intervals instead of closed intervals.

Usually I was kinda frustrated at the "bad" API I had to work with.

https://us.metamath.org/mpeuni/lcmineqlem12.html#128

This was in my opinion too complicated. And it also used the derivative of the closed interval.

So it might be that both versions are necessary. However I think that you may be able to get away with just open intervals. I guess I'll figure it out

Reply:
> https://us.metamath.org/mpeuni/ftc2.html

Yes, I had looked at that theorem. However, if there is a theorem xxx, then writing in the comment of theorem yyy "xxx version" is not enlightening. I genuinely do not know what it is meant to mean.

Reply:
It just meant that the theorem I've formalised is in a form where there it is significantly easier to use with the fundamental theorem of calculus. That's it.